Page/LoginPage.robot

```diff
@@ -1,14 +1,30 @@
 *** Settings ***
 Library           Selenium2Library
 Resource          ../globalConfig/testEnv.robot
+Resource          NavigatorPage.robot
 
 *** Variables ***
-${USER_NAME_INPUT_TEXT_LOCATOR}    j_username
-${PASSWORD_INPUT_TEXT_LOCATOR}    j_password
+${LOGIN_ERROR_MESSAGE_LOCATOR}    //font[contains(text(), 'Incorrect login or password.')]
+${USER_INPUT_LOCATOR}    //input[@id='j_username']
+${PASSWORD_INPUT_LOCATOR}    //input[@name='j_password']
+${LOGIN_BUTTON_LOCATOR}    //button[contains(text(),'Login')]
+${LOGIN_PAGE_TOP_LOGO_IMAGE_LOCATOR}    //img[@src='images/logo.gif?b=']
 
 *** Keywords ***
 Login
-    Open Browser    ${CMP_ADRESS}
-    Input Text    ${USER_NAME_INPUT_TEXT_LOCATOR}    ${ADMIN_LOGIN}
-    Input Text    ${PASSWORD_INPUT_TEXT_LOCATOR}    ${ADMIN_PASSWORD}
+    [Arguments]    ${login}    ${password}
+    Input Text    ${USER_INPUT_LOCATOR}    ${login}
+    Input Text    ${PASSWORD_INPUT_LOCATOR}    ${password}
     Submit Form
+
+Is Error Message Visible
+
+Wait For Login Error Message
+    Wait Until Element Is Visible    ${LOGIN_ERROR_MESSAGE_LOCATOR}    ${SELENIUM_DEFAULT_TIMEOUT}
```

Comment:
No error message presented. See Wait Until Element Is Visible of Selenium2Library docs to set a message if keyword fails. Logging is useful to identify an issue